f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -497,6 +502,13 @@ def connect_driver(self, context):
         if self.lbdriver:
             self.lbdriver.connect()
 
+    @periodic_task.periodic_task(spacing=PERIODIC_TASK_INTERVAL)
+    def refresh_esd(self, context):
+        """Refresh ESD files."""
+        if self.lbdriver.esd and self.conf.esd_auto_refresh:
```

Comment:
Could we rename `self.lbdriver.esd` to `self.lbdriver.esdHelper` or so?

Reply:
Why do you think of using `esdHelper` this name?

Reply:
First glance at this name, I thought it's the ESD entities. In fact, it's a (ESD)processor.